src/ngraph/op/util/broadcasting.cpp

```diff
@@ -0,0 +1,296 @@
+//*****************************************************************************
```

Comment:
This seems to duplicate what's already in `src/ngraph/builder/autobroadcast.[ch]pp`. If there's any functionality you need that's not covered by those builders, could you incorporate that functionality into them and remove this?

Reply:
I actually didn't realize ONNX had its own implementation either... two's company, three's a crowd, I guess ;)